test/hack/checkLabels/main.go

```diff
@@ -13,6 +13,14 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+/*
+* GLOSSARY
+* https://github.com/giantswarm/prometheus-rules/#inhibitions
+* am_sourceMatchers and am_targetMatchers are the labels defined in the alertmanager config file
+* sourceLabels are the labels defined in the alerting rules from which originate the inhibitions
+* 	--> For example, 'scrape_timeout' is the source label for the 'cancel_if_scrape_timeout' inhibition label
+* cancelLabels is another name for the inhibition labels (for example : 'cancel_if_scrape_timeout')
+**/
 const output = "../output"
 const target = "prometheus-rules/templates/alerting-rules"
```

Comment:
Also this variable name is confusing as we are dealing with a lot of target labels in here. Maybe rename this to something like:
```suggestion
const alertRulesPath = "prometheus-rules/templates/alerting-rules"
```

Reply:
Done